ui/raidboss/popup-text.js

```diff
@@ -602,39 +602,43 @@ export class PopupText {
   }
 
   OnLog(e) {
+    // This could concievably be determined based on the line's contents as well, but
```

Comment:
```suggestion
    // This could conceivably be determined based on the line's contents as well, but
```

I think line timings get weird because of the dropped milliseconds on `00` lines.  So, it's a little bit more reliable to have an independent timer, I think.

Reply:
Yeah, plus this has the added benefit of exactly following the old behavior whereas if it pulled from the log line it could cause weird behavior if there was lag in the line parsing or whatever.

Reply:
The suggested change didn't fix both misspellings so I `--amend`-ed the commit to fix them both instead.

Reply:
Maybe I'm not seeing it, but did you change it?

Reply:
Yes, it's changed and force pushed.

Reply:
Or not. Now it should be. I really need to fix my vscode git integration.